recipes/libmatio/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+./configure --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+
+if [[ `uname` == 'Darwin' ]];
+then
+	eval DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib make check
+else
+	make check
```

Comment:
I think we still need something similar for Linux using `LD_LIBRARY_PATH` instead.